docs/how-to/gpu-enabled-mpi.rst

```diff
@@ -0,0 +1,260 @@
+***************************************************************************************************
+GPU-enabled MPI
+***************************************************************************************************
+
+The Message Passing Interface (`MPI <https://www.mpi-forum.org>`_) is a standard API for distributed
+and parallel application development that can scale to multi-node clusters. To facilitate the porting of
+applications to clusters with GPUs, ROCm enables various technologies. You can use these
```

Comment:
Looks good.

Missing 'to' in 'to add'? 

>You can use these technologies add GPU pointers to MPI calls and enable ROCm-aware MPI libraries to deliver optimal
performance for both intra-node and inter-node GPU-to-GPU communication.